ethcore/sync/src/block_sync.rs

```diff
@@ -327,6 +327,9 @@ impl BlockDownloader {
 					if self.limit_reorg && best > last && (last == 0 || last < oldest_reorg) {
 						trace_sync!(self, "No common block, disabling peer");
 						return Err(BlockDownloaderImportError::Invalid);
+					} else if !self.limit_reorg {
```

Comment:
So this ensures that it's downloading _OldBlocks_, right? Can't we use a more specific name or `fn`?